vis-lua.c

```diff
@@ -2735,7 +2735,6 @@ void vis_lua_init(Vis *vis) {
 		const char *name;
 	} textobjects[] = {
 		{ VIS_TEXTOBJECT_INNER_WORD, "text_object_word" },
-		{ VIS_TEXTOBJECT_INNER_LONGWORD, "text_object_longword" },
```

Comment:
I think this wrong, that was the problem I had with my rebase of the patch. vis-lua.c should not be changed at all, IMHO.

Reply:
Yeah, this looks suspicious. I faintly remember that I used this in an earlier version of the code and then stopped using it. Apparently after I added a commit to remove this one which doesn't seem right to me now.

I have reverted this change.